docs/featurestore/featurestore.rst

```diff
@@ -834,16 +834,20 @@ Follow the steps described in `Connecting from Amazon SageMaker`_ for setting up
 
 The feature store library needs to be installed to connect to it. In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries* and then *Install New*. As *Library Source* choose *PyPI* and fill in *hopsworks-cloud-sdk* into the *Package* field.
 
-**Mounting a bucket for storing certificates**
+**Creating a DBFS folder to store certificates**
+
+Hopsworks relies on certificates being available in the Databricks cluster to connect to some services inside Hopsworks. To ensure that these certificates can be distributed to all nodes in a Databricks cluster, Hopsworks relies on them being stored in the Databricks file system.
+For this purpose, you should create a cert folder in DBFS. This can be done by running the following in a Databricks notebook connected to the prepared cluster::
+
+    dbutils.fs.mkdirs("dbfs:/certs/")
```

Comment:
We should also mention the refresh then and maybe link the official documentation.

Reply:
I think the refresh is only needed if we mount our own bucket, but as we use the one created by databricks I don't think we need the refresh. I'll add a link to the dbfs doc.

Reply:
Right I didn't think about that.